tensormap-client/src/components/DragAndDropCanvas/Sidebar.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const Sidebar =  () => {
+  const onDragStart = (event, nodeType) => {
```

Comment:
Here we have used ES6 function and I see above at some places we have used normal functions, I think to maintain code consistency, we should stick with one. 
Usually it is always preferred to go with ES6+
What are your views?

Reply:
Agreed. Will go ahead with ES6

Reply:
Cool  @Wenuka19 😄